install/changelog.md

```diff
@@ -52,6 +57,100 @@ title: "Change Log"
 
 > See [known issues](../reference/known-issues.html) for currently known issues and workarounds.
 
+## v0.15.0 {#v150}
+
+Released on August 13, 2018
+
+In addition to the 0.15.0 CLI release, we've released new versions of all of our packages.
+
+### Pulumi CLI
+
+This release brings many large improvements to the Pulumi CLI. Major features include:
+
+#### Kubernetes
```

Comment:
@hausdorff Can you look this over and/or suggest edits?

Reply:
This obviously doesn't belong in the CLI section ... @lukehoban How do we elevate this to a top-level thing? I don't want to bury the lede on our most exciting part of this release!

Reply:
Once we've have all the content here - I'll reorganize a bit.  We can pull a few of the "big" things up to the top.